test-cases/gutenberg/spacer.md

```diff
@@ -74,6 +74,10 @@ Expected look:
 ##### TC005
 
 ### Settings: Height range extends if Spacer comes from the web is higher than 500px
+<details>
+ <summary>Deprecation note</summary>
+  Setting height bigger than 500px is no longer available on the web, however, we still need to do this test because of backward compatibility. In order to test it, please use the code editor mode to change the height of the spacer.
```

Comment:
```suggestion
  Setting height bigger than 500px is no longer available on the web, however, we still need to do this test because of backward compatibility. In order to test it, please use the html editor mode to change the height of the spacer.
```

Thanks for the deprecation note, just added a small suggestion since it is called "HTML mode" inside the app. 

I think we can remove this test case after some time.